kimchi/src/circuits/expr.rs

```diff
@@ -582,11 +582,11 @@ pub enum FeatureFlag {
 
 impl FeatureFlag {
     fn is_enabled(&self) -> bool {
-        todo!("Handle features")
+        true
```

Comment:
@mrmr1993 I don't think I ever got an answer of what to do here.  Currently, this just unconditionally enables all features (since I assume the kimchi expression is correct).  An inspection of the `IfFeature` branch of `value_`/`value` indicates this shouldn't be broken, but I don't know whether to revert this or whatever.